### PR TITLE
dyndns widget - fix cached IP status colors to standard values

### PR DESCRIPTION
**Pull request by opnsenseuser**
…/red

the ip color of the dyndns widget is now automatically adjusted for each theme by using standard classes like (text-success or text-warning). This results in a nicer overall picture and does not use another color value of green or red.

f.e. -> using theme colors
grafik

### CODE REVIEW

**Comment by opnsenseuser** [comment]
@fichtner i know there are much more important things to fix, but what is your opinion on this?

Regards Rene

**Comment by fabianfrz** [comment]
@opnsenseuser I would suggest you to increase the plugin version as well (you can do that directly online).

File: dns/dyndns/src/www/widgets/widgets/dyn_dns_status.widget.php [hunk 1]
```diff
@@ -64,16 +64,16 @@
             $cached_ip_s = explode('|', $fdata);
             $cached_ip = $cached_ip_s[0];
             echo sprintf(
-                '<font color="%s">%s</font>',
-                $ipaddr != $cached_ip ? 'red' : 'green',
+                '<font class="%s">%s</font>',
```
Comment (fichtner) [inline dns/dyndns/src/www/widgets/widgets/dyn_dns_status.widget.php]:
```suggestion
                '<span class="%s">%s</span>',
```

File: dns/dyndns/src/www/widgets/widgets/dyn_dns_status.widget.php [hunk 2]
```diff
@@ -64,16 +64,16 @@
...
+                $ipaddr != $cached_ip ? 'text-warning' : 'text-success',
                 htmlspecialchars($cached_ip)
             );
         } elseif (!empty($fdata6)) {
             $cached_ipv6_s = explode('|', $fdata6);
             $cached_ipv6 = $cached_ipv6_s[0];
             echo sprintf(
-                '<font color="%s">%s</font>',
-                $ipv6addr != $cached_ipv6 ? 'red' : 'green',
+                '<font class="%s">%s</font>',
```
Comment (fichtner) [inline dns/dyndns/src/www/widgets/widgets/dyn_dns_status.widget.php]:
```suggestion
                '<span class="%s">%s</span>',
```

**Comment by opnsenseuser** [comment]
@fichtner sorry - didn´t also know that i have to confirm these changes. hope you can merge it soon. thx . regards rené

**Comment by fichtner** [comment]
merged, thanks!

**Pull request merged**